spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/sql/TestUnpartitionedWrites.java

```diff
@@ -18,12 +18,4 @@
  */
 package org.apache.iceberg.spark.sql;
 
-import java.util.Map;
-
-public class TestUnpartitionedWrites extends UnpartitionedWritesTestBase {
-
-  public TestUnpartitionedWrites(
-      String catalogName, String implementation, Map<String, String> config) {
-    super(catalogName, implementation, config);
-  }
-}
+public class TestUnpartitionedWrites extends UnpartitionedWritesTestBase {}
```

Comment:
As well as `TestPartitionedWrites`, I will remove this test if this is not required.

Reply:
it is required. I don't see a reason to remove this test class